[2024-04-05 19:36:14,231] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.extrai_dados scheduled__2023-12-25T00:00:00+00:00 [queued]>
[2024-04-05 19:36:14,235] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.extrai_dados scheduled__2023-12-25T00:00:00+00:00 [queued]>
[2024-04-05 19:36:14,235] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-04-05 19:36:14,235] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2024-04-05 19:36:14,235] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-04-05 19:36:14,253] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): extrai_dados> on 2023-12-25 00:00:00+00:00
[2024-04-05 19:36:14,255] {standard_task_runner.py:52} INFO - Started process 725840 to run task
[2024-04-05 19:36:14,257] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'extrai_dados', 'scheduled__2023-12-25T00:00:00+00:00', '--job-id', '104', '--raw', '--subdir', 'DAGS_FOLDER/extrai_dados_climaticos.py', '--cfg-path', '/tmp/tmppa4xdfb0', '--error-file', '/tmp/tmp2u3o_23t']
[2024-04-05 19:36:14,258] {standard_task_runner.py:80} INFO - Job 104: Subtask extrai_dados
[2024-04-05 19:36:14,298] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.extrai_dados scheduled__2023-12-25T00:00:00+00:00 [running]> on host TEC017860.
[2024-04-05 19:36:14,348] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=extrai_dados
AIRFLOW_CTX_EXECUTION_DATE=2023-12-25T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-12-25T00:00:00+00:00
[2024-04-05 19:36:14,976] {python.py:173} INFO - Done. Returned value was: None
[2024-04-05 19:36:14,979] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=extrai_dados, execution_date=20231225T000000, start_date=20240405T223614, end_date=20240405T223614
[2024-04-05 19:36:15,008] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-04-05 19:36:15,017] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
